### ISSUE TITLE
Add feature to run the trace command and see the output

### ISSUE THREAD

**Issue by reenberg** [streetsidesoftware/vscode-spell-checker#2976]
As stated in #2110 by @Jason3S:

> The VSCode Code Spell Checker extension does not currently have a trace command, but that could be useful.

It would be beneficial for non Node.js users, who most likely don't have cSpell installed in their environment, to easily use the trace method inside VS Code, to debug words.   Especially as additional dictionaries might be installed/enabled which could be non trivial for people to setup externally, compared to a functional VS Code setup.

Also perhaps, if it was also available using the command palette, then it could use the current open file as reference and set the --language-id accordingly, to aide and give more precise results.

**Comment by reenberg** [comment]
Currently the extension is VS Code doesn't flag the word "linters" in my open Markdown file "development.md".
However if I lint it on the command line, then it marks it as unknown.

If I trace it on the command line, then it shows that only the US dictionary matches the word "linters", and it also shows that as disabled, thus it should correctly be flagged as misspelled.  

I'm just unsure why it is not flagged in VS Code, then. I have v3.0.1 of the "Code Spell Checker" installed. And in the Spell Checker Preferences dialogue, it seems to show perfectly that only the "en-GB" dictionary is enabled, under the File tab (note that the list of dictionaries is not sorted in this view), the rest of the dictionaries match the ones from the command line cSpell trace.  

So I'm kind of at a loss why it is accepted in VS Code but not at the command line, unless something was changed in one of the dictionaries between the version in VS Code and on the command line.

```shell
$ npx cspell --version
8.1.3

$ npx cspell lint docs/about/development.md 
1/1 ./docs/about/development.md 578.52ms X
./docs/about/development.md:217:41 - Unknown word (linters)

$ npx cspell trace --language-id Markdown linte
rs
Word    F Dictionary                Dictionary Location
linters - [flagWords]*              From Settings `flagWords`
linters - [ignoreWords]*            From Settings `ignoreWords`
linters - [suggestWords]*           From Settings `suggestWords`
linters - [words]*                  From Settings `words`
linters - aws*                      node_modules/@cspell/dict-aws/dict/aws.txt
linters - companies*                node_modules/@cspell/dict-companies/dict/companies.txt
linters - computing-acronyms*       node_modules/@cspell/.../dict/computing-acronyms.txt
linters - cryptocurrencies*         node_modules/@cspell/.../dict/cryptocurrencies.txt
linters * en_us                     node_modules/@cspell/dict-en_us/en_US.trie.gz
linters - en-gb*                    node_modules/@cspell/dict-en-gb/en_GB.trie.gz
linters - en-gb-common-misspelling* node_modules/@cspell/.../dict-en-gb.yaml
linters - filetypes*                node_modules/@cspell/dict-filetypes/filetypes.txt.gz
linters - html*                     node_modules/@cspell/dict-html/dict/html.txt
linters - html-symbol-entities*     node_modules/@cspell/.../entities.txt.gz
linters - npm*                      node_modules/@cspell/dict-npm/dict/npm.txt
linters - project-words*            project-words.txt
linters - public-licenses*          node_modules/@cspell/.../public-licenses.txt.gz
linters - softwareTerms*            node_modules/@cspell/.../dict/softwareTerms.txt
linters - web-services*             node_modules/@cspell/.../dict/webServices.txt
```

**Comment by Jason3S** [comment]
@reenberg,

Thank you for the request. It is not easy to figure out why something isn't considered an error.

**Comment by reenberg** [comment]
@Jason3S , can't wait to try it out.  Do you have an idea of when you will make the next release?
